### PR TITLE
[TS-209] Implement UI for notification dropdown

### PR DESCRIPTION
**Pull request by Ming424**
Wait until [TS-51] merged
There is one asset from there
https://youtu.be/2t1Xxf1t8Gw

[TS-51]: https://tradeshare490.atlassian.net/browse/TS-51?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by AlyaNaseer** [comment]
Looks cool! You've got one security issue though, it's because you used v-html, it doesn't like it since people can inject content that way. You can look at reworking it so that it is passed via {{ }}

**Comment by Ming424** [comment]
> Looks cool! You've got one security issue though, it's because you used v-html, it doesn't like it since people can inject content that way. You can look at reworking it so that it is passed via {{ }}.
> 
> Also, are we not doing read/unread notifications?

I notice the security issue, trying to find alternative for them.
I will add a new field for read/unread

**Comment by Ming424** [comment]
Update:
+ Hover animation
+ Logic for read/unread
+ Pane header
image

File: src/components/NavBar/NavBar.vue [hunk 1]
```diff
@@ -59,9 +59,60 @@
...
+                  </v-col>
+                </v-list-item>
+              </v-list>
+              <v-list
+                v-else
+                three-line
+                class="NotificationList"
+              >
+                <v-subheader class="subheader font-weight-bold mt-1 black--text">
+                  Lastest Notification
```
Comment (itsShaVawn) [inline src/components/NavBar/NavBar.vue:99]: Add s to notification

File: src/components/NavBar/NavBar.vue [hunk 2]
```diff
@@ -88,7 +143,26 @@ export default {
       isLoading: false,
       searchModel: null,
       search: null,
-      searchQueue: []
+      searchQueue: [],
+      notifItems: [
+        { avatar: 'https://cdn.vuetifyjs.com/images/lists/1.jpg', message: 'Ali Connors I\'ll be in your neighborhood doing errands this weekend. Do you want to hang out?', date: '3/12/2021, 7:14:33 PM', readed: false },
```
Comment (itsShaVawn) [inline src/components/NavBar/NavBar.vue:148]: try using back ticks and `${}` to inject dynamic data into the string

**Review by itsShaVawn — CHANGES_REQUESTED** [review]
Would be better to have the "read" on click instead of on hover

File: src/components/NavBar/NavBar.vue (hunk 1 above)
Comment (AlyaNaseer) [reply src/components/NavBar/NavBar.vue:99]: And also remove the "s" from Lastest --> Latest

**Comment by AlyaNaseer** [comment]
> Would be better to have the "read" on click instead of on hover

I agree

File: src/components/NavBar/NavBar.vue [hunk 3]
```diff
@@ -88,7 +143,26 @@ export default {
...
+        { message: 'to Alex, Scott, Jennifer: Wish I could come, but I\'m out of town this weekend.', date: '1/11/2019, 7:14:33 PM', readed: false },
```
Comment (AlyaNaseer) [inline src/components/NavBar/NavBar.vue:149]:
```suggestion
        { message: 'to Alex, Scott, Jennifer: Wish I could come, but I\'m out of town this weekend.', date: '1/11/2019, 7:14:33 PM', read: false },
```
I'd have read/unread instead ("readed" isn't a word ;) )

**Review by AlyaNaseer — CHANGES_REQUESTED** [review]

File: src/components/NavBar/NavBar.vue (hunk 3 above)
Comment (Ming424) [reply src/components/NavBar/NavBar.vue:149]: its fixed!

File: src/components/NavBar/NavBar.vue (hunk 2 above)
Comment (Ming424) [reply src/components/NavBar/NavBar.vue]: I am using a new data field call subject for rendering
image
image

File: src/components/NavBar/NavBar.vue (hunk 1 above)
Comment (Ming424) [reply src/components/NavBar/NavBar.vue]: fixed
image

**Comment by Ming424** [comment]
> Would be better to have the "read" on click instead of on hover

Its click triggered now

**Review by itsShaVawn — APPROVED** [review]
Nice 👍🏼

**Review by AlyaNaseer — APPROVED** [review]

**Pull request merged**